libs/sdk-ui-dashboard/src/presentation/widget/insight/configuration/InsightDrillConfigPanel/InsightDrillConfigPanel.tsx

```diff
@@ -25,12 +25,23 @@ export const InsightDrillConfigPanel: React.FunctionComponent<IDrillConfigPanelP
         drillConfigItems,
         originSelectorItems,
         isOriginSelectorVisible,
+        isLoaded,
         onChangeItem,
         onOriginSelect,
         onSetupItem,
         onDeleteItem,
     } = useInsightDrillConfigPanel({ widgetRef });
 
+    const originSelector = useMemo(() => {
```

Comment:
Does this needs to be wrapped in useMemo? I think that common conditions in JSX should be enough.

Reply:
fiexd